homeassistant/components/doorbird.py

```diff
@@ -0,0 +1,45 @@
+"""Support for a DoorBird video doorbell."""
+
+import logging
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST, CONF_USERNAME, CONF_PASSWORD
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['DoorBirdPy==0.0.4']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'doorbird'
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_HOST): cv.string,
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string
+    })
+}, extra=vol.ALLOW_EXTRA)
+
+
+def setup(hass, config):
+    """Set up the DoorBird component."""
+    ip = config[DOMAIN].get(CONF_HOST)
+    username = config[DOMAIN].get(CONF_USERNAME)
+    password = config[DOMAIN].get(CONF_PASSWORD)
+
+    from doorbirdpy import DoorBird
+    device = DoorBird(ip, username, password)
+    status = device.ready()
+
+    if status[0]:
+        _LOGGER.info("Connected to DoorBird at %s as %s", ip, username)
+        hass.data[DOMAIN] = device
```

Comment:
I'm not sure what this means. Do I define a new constant called `DATA_DOORBIRD` and use that here? But what would its value be? I'm using the Ring component as an example, and this is what is used there.